_includes/header.html

```diff
@@ -2,7 +2,13 @@
 
   <div class="wrapper">
 
-    <a class="site-title" href="{{ site.baseurl }}/">{{ site.title | escape }}</a>
+    <a class="site-title" href="{{ site.baseurl }}/">
+      {% if site.logo %}
+        <img src="/assets/{{ site.logo }}" class="site-logo" width="150" height="auto">
```

Comment:
We copy directly into `/assets` in the `_site` folder which might affect the site as far as we would not preserve the images and they would always be recopied and in development if a file in `_assets` conflicts with a file in `/assets` we would win and overwrite their copy, though this would only be a problem in development or if the user explicitly disabled SHA's.